unit/goto-programs/goto_trace_output.cpp

```diff
@@ -0,0 +1,4 @@
+//
```

Comment:
Nit: dunno how this file add ended up in this commit, but it probably doesn't want to be here!